openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx

```diff
@@ -167,64 +173,71 @@ const DataInsightSummary: FC<Props> = ({ chartFilter, onScrollToChart }) => {
         </Typography.Title>
       }>
       <Row data-testid="summary-card-content" gutter={[16, 16]}>
-        {/* summary of entity charts */}
-        {entitiesSummaryList.map((summary) => (
-          <Col
-            className="summary-card-item"
-            data-testid={`summary-item-${summary.id}`}
-            key={summary.id}
-            span={6}
-            onClick={() => onScrollToChart(summary.id)}>
-            <Typography.Text className="data-insight-label-text">
-              {summary.label}
-            </Typography.Text>
-            <Typography className="font-semibold text-2xl m--ml-0.5">
-              {summary.latest}
-              {summary.id.startsWith('Percentage') ? '%' : ''}
-            </Typography>
-          </Col>
-        ))}
-
-        {/* summary for web charts */}
-        {webSummaryList.map((summary) => (
-          <Col
-            className="summary-card-item"
-            data-testid={`summary-item-${summary.id}`}
-            key={summary.id}
-            span={6}
-            onClick={() => onScrollToChart(summary.id)}>
-            <Typography.Text className="data-insight-label-text">
-              {summary.label}
-            </Typography.Text>
-            <Typography className="font-semibold text-2xl m--ml-0.5">
-              {summary.latest}
-              {summary.id.startsWith('Percentage') ? '%' : ''}
-            </Typography>
-          </Col>
-        ))}
-
-        {/* summary of most active user */}
-        {mostActiveUser && mostActiveUser.userName && (
-          <Col
-            data-testid={`summary-item-${DataInsightChartType.MostActiveUsers}`}
-            key={DataInsightChartType.MostActiveUsers}
-            span={6}>
-            <Typography.Text className="data-insight-label-text d-block">
-              {t('label.most-active-user')}
-            </Typography.Text>
-            <UserPopOverCard userName={mostActiveUser.userName}>
-              <Space>
-                <ProfilePicture
-                  id=""
-                  name={mostActiveUser.userName}
-                  type="circle"
-                />
-                <Link to={getUserPath(mostActiveUser.userName)}>
-                  {mostActiveUser.userName}
-                </Link>
-              </Space>
-            </UserPopOverCard>
-          </Col>
+        {tab === DataInsightTabs.DATA_ASSETS && (
```

Comment:
Eventually we should update to AntD Tabs

Reply:
We are not using the tab component, we have deep link support. so basically `tab` is from the params and based on the params we are rendering the components.